TextileNode.ts

```diff
@@ -102,7 +102,7 @@ export default {
     return overview
   },
 
-  async addThread (name: TT.ThreadName): Promise<TT.Thread> {
+  async addThread (name: string): Promise<TT.Thread> {
```

Comment:
At this point, the `name` really is just a `string` because it's coming from the user input.